recipe/meta.yaml

```diff
@@ -99,9 +99,10 @@ outputs:
         - test -f $PREFIX/targets/{{ target_name }}/include/cusparse.h                         # [linux]
         - test -L $PREFIX/targets/{{ target_name }}/lib/libcusparse.so                         # [linux]
         - test -f $PREFIX/targets/{{ target_name }}/lib/stubs/libcusparse.so                   # [linux]
-        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran.c         # [linux]
-        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran.h         # [linux]
-        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran_common.h  # [linux]
+        # These `src` files are only available on Linux x86_64 (not aarch64 or ppc64le)
+        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran.c         # [linux64]
+        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran.h         # [linux64]
+        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran_common.h  # [linux64]
```

Comment:
Noted this change in issue ( https://github.com/conda-forge/libcusparse-feedstock/issues/2 ) for tracking